packages/app-root/src/app/layout.js

```diff
@@ -2,8 +2,23 @@ import RootLayout from '@/components/RootLayout'
 import StyledComponentsRegistry from './style-registry'
 
 export const metadata = {
-  title: 'Zooniverse',
-  description: 'People-powered Research'
+  title: {
+    template: '%s | Zooniverse',
```

Comment:
This means when a user visits `/about`, the `<head>` title becomes "About | Zooniverse".